java/lang/information_leakage.yml

```
@@ -0,0 +1,55 @@
+patterns:
+  - pattern: |
+      try {
+        $<...>
+      } catch($<EXCEPTION> $<EXCEPTION_VAR:identifier>) {
+        $<!>$<EXCEPTION_VAR>.printStackTrace();
+      }
+    filters:
+      - either:
+          - variable: EXCEPTION
+            values:
+              - Throwable
+              - Exception
+              - Error
+              - OutOfMemoryError
+              - StackOverflowError
+          - variable: ERROR
+            regex: ^(java.)?(io.)?FileNotFoundException
```

Just thinking about these package names more deeply, Java types are either fully qualified or just the type name, so you don't need to make each part of the package name optional. Also we should be escaping the dots (need to check the escaping works ok with YAML parsing):
```suggestion
            regex: ^(java\.io\.)?FileNotFoundException
```

---

Here's the general PR: https://github.com/Bearer/bearer-rules/pull/109